extensions/amp-form/0.1/amp-form.js

```diff
@@ -183,22 +186,24 @@ export class AmpForm {
       return;
     }
 
-    // Validity checking should always occur, novalidate only circumvent
-    // reporting and blocking submission on non-valid forms.
-    const isValid = checkUserValidityOnSubmission(this.form_);
-    if (this.shouldValidate_ && !isValid) {
-      if (opt_event) {
-        opt_event.stopImmediatePropagation();
-        opt_event.preventDefault();
+    if (isCheckValiditySupported(this.win_.document)) {
+      // Validity checking should always occur, novalidate only circumvent
+      // reporting and blocking submission on non-valid forms.
+      const isValid = checkUserValidityOnSubmission(this.form_);
+      if (this.shouldValidate_ && !isValid) {
```

Comment:
No we can't since that might signal that would make the form invalid all the time and hence prevent the user from submitting the form ever, so the best thing to do here is just let the form be submitted as is if we can't validate it